<launch>
	<node pkg = "rosaria" type = "RosAria" name = "RosAria">
		<param name="port" value="/dev/ttyS0" type="string"/>
	</node>

	<node pkg = "rosaria_client" type = "pose_cmd" name = "RosAria_pose_cmd" launch-prefix = "xterm -e"/>

</launch>
